### ISSUE TITLE
Dinosaurs don't seem to drink for me?

### ISSUE THREAD

**Issue by epicslayer999** [JurassiCraftTeam/JurassiCraft2#208]
I don't know, but I have never witnessed dinosaurs drinking. Do they have to be labeled as "thirsty" in the InGen Field Guide for them to seek water and drink? If this is the case, then my dinosaurs are never thirsty because they re-gain thirst when it rains. And is the raining thing intended?

**Comment by gegy1000** [comment]
Yes, their thirst is restored whenever it rains, and they only drink when the thirsty modifier is shown in the InGen Field Guide.
